autobuild/windows/autobuild_windowsinstaller_2_build.ps1

```diff
@@ -27,4 +28,11 @@ if (("$jamulus_project_path" -eq $null) -or ("$jamulus_project_path" -eq "")) {
 
 echo "Build installer..."
 # Build the installer
-powershell "$jamulus_project_path\windows\deploy_windows.ps1" "C:\Qt\5.15.2"
+if ($BuildOption -ne "")
+{
+    powershell "$jamulus_project_path\windows\deploy_windows.ps1" "C:\Qt\5.15.2" -BuildOption $BuildOption
+}
+else
+{
+    powershell "$jamulus_project_path\windows\deploy_windows.ps1" "C:\Qt\5.15.2"
+}
```

Comment:
Not sure about Windows/PowerShell behavior, but if passing an empty parameter is an option (maybe needs quoting?) the logic could be simplified?

```suggestion
powershell "$jamulus_project_path\windows\deploy_windows.ps1" "C:\Qt\5.15.2" -BuildOption "$BuildOption"
```

Reply:
Passing an empty parameter is not a big issue, however specify an option without a value (even empty) seems to be an issue.
To keep it consistent in the scripts and allowing a "build" without specifying an option, I have set it up like this.

Reply:
Hrm, too bad. :(
Then it will probably have to stay like this.